tests/Builder/FormContractorTest.php

```diff
@@ -88,9 +89,13 @@ public function testDefaultOptionsForSonataFormTypes()
         ];
         $collectionTypes = [
             'sonata_type_collection',
-            CollectionType::class,
+            DeprecatedCollectionType::class,
         ];
 
+        if (class_exists(CollectionType::class)) {
```

Comment:
shouldn't this have a `\`?

Reply:
I just wanted to say that it is strange that @flintci isn't complaining and now I see it isn't even in the list, cc @Soullivaneuh